### PR TITLE
Added Delete comment function to each comment

### PR DESCRIPTION
**Pull request by lattai**
See at : http://leahattai-step-2020.appspot.com/comments.html

### CODE REVIEW

File: portfolio/src/main/webapp/comments-script.js [hunk 1]
```diff
@@ -84,3 +85,19 @@ function getMaxComments (){
...
 }
+
+function createDeleteButton(comment) {
+    const deleteButton = document.createElement('button');
+    deleteButton.innerText = 'Delete';
+    deleteButton.addEventListener('click', () => {
+        deleteComment(comment);
+    });
+    return deleteButton
+}
```
Comment (menynexi) [inline portfolio/src/main/webapp/comments-script.js:96]: Wouldn't it be best top call it commentDeleteButton

File: portfolio/src/main/java/com/google/sps/servlets/Comment.java [hunk 2]
```diff
@@ -29,6 +29,7 @@
     private final String message;
     private final long timestamp;
     private Key key;
+    private long id;
```
Comment (menynexi) [inline portfolio/src/main/java/com/google/sps/servlets/Comment.java:32]: What does the variable id refer to?

File: portfolio/src/main/java/com/google/sps/servlets/Comment.java [hunk 3]
```diff
@@ -54,6 +55,10 @@ public Key getKey(){
         return key;
     }
 
+    public long getId(){
```
Comment (menynexi) [inline portfolio/src/main/java/com/google/sps/servlets/Comment.java:58]: public long getId()  { missing a space

File: portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+@WebServlet("/delete-comment")
+
+public class DeleteCommentServlet extends HttpServlet {
+
+    private static String maxComments;
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long id = Long.parseLong(request.getParameter("id"));
```
Comment (DirtGrubDylan) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java:47]: String as variable

File: portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+    Key taskEntityKey = KeyFactory.createKey("Comment", id);
```
Comment (DirtGrubDylan) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java:49]: Same here

**Pull request merged**